src/Middleware/WhoopsErrorResponseGenerator.php

```diff
@@ -91,11 +91,18 @@ private function prepareWhoopsHandler(ServerRequestInterface $request, PrettyPag
     {
         $uri = $request->getAttribute('originalUri', false) ?: $request->getUri();
         $request = $request->getAttribute('originalRequest', false) ?: $request;
+        
+        $serverParams = $request->getServerParams();
+        $scriptName = '';
+        
+        if (isset($serverParams['SCRIPT_NAME'])) {
+            $scriptName = $serverParams['SCRIPT_NAME'];
+        }
```

Comment:
This can be cleaned up to:
```
$serverParams = $request->getServerParams();
$scriptName = (isset($serverParams['SCRIPT_NAME'])) ? $serverParams['SCRIPT_NAME'] : '';
```
It'll be nice when ZE is pinned to PHP 7.1 and we can use null coalesce operators...